tests/llmcompressor/transformers/autoround/test_autoround_oneshot.py

```diff
@@ -39,6 +39,29 @@
     },
 )
 
+rtn_recipe_modifier = AutoRoundModifier(
+    ignore=["lm_head"],
+    iters=0,
+    config_groups={
+        "group_0": QuantizationScheme(
+            targets=["Linear"],
+            weights=QuantizationArgs(num_bits=8, strategy="group", group_size=128),
+            input_activations=QuantizationArgs(num_bits=8, type="float", strategy="tensor", dynamic=True),
+        )
+    },
+)
+
+act_quant_recipe_modifier = AutoRoundModifier(
```

Comment:
How about renaming this variable to include the quantization scheme, such as `w8a8_fp8_quant_recipe_modifier`?

And please add the UT for the schemes used in the examples.

Reply:
updated